sphinx_markdown_parser/markdown_parser.py

```diff
@@ -161,6 +161,8 @@ def isblocklevel(self, html):
         return False
 
     def walk_markdown_ast(self, node):
+        if isinstance(node, str):
```

Comment:
maybe we should check what we expect node to be and not one case what it might be instead.

e.g.
```
if not isinstance(node, nodes.Node)
```

http://code.nabla.net/doc/docutils/api/docutils/nodes/docutils.nodes.Node.html#docutils.nodes.Node

Reply:
another interesting question might be why it is a `str` instead of a node and avoid this earlier?